src/types.ts

```diff
@@ -74,15 +73,15 @@ export interface Position {
   rowIdx: number;
 }
 
-export interface FormatterProps<TRow = any, TSummaryRow = any> {
+export interface FormatterProps<TRow, TSummaryRow = unknown> {
```

Comment:
I think it's fair to at least ask for `TRow`. Users can still use `<unknown>` or `<any>` if they really want to.